package.json

```diff
@@ -29,21 +26,17 @@
     "filereader-stream": "2.0.0",
     "hmda-file-parser": "1.18.0",
     "react": "16.4.2",
+    "react-app-polyfill": "0.1.3",
     "react-dom": "16.4.2",
     "react-dropzone": "5.1.0",
     "react-redux": "5.0.7",
     "react-router-dom": "4.3.1",
-    "react-scripts": "2.0.1",
+    "react-scripts": "2.0.3",
     "redux": "4.0.0",
     "redux-logger": "3.0.6",
     "redux-thunk": "2.3.0",
     "split2": "3.0.0"
   },
   "homepage": "/tools",
-  "browserslist": [
-    ">0.2%",
-    "not dead",
-    "not ie <= 11",
-    "not op_mini all"
-  ]
+  "browserslist": [">0.2%", "not dead", "not ie <= 11", "not op_mini all"]
```

Comment:
Shouldn't this be `not ie < 11` rather than `<=`?

Reply:
Indeed it should. Fixed in a6e19b5.